charts/caraml-authz/templates/deployment.yaml

```diff
@@ -57,6 +57,12 @@ spec:
           value: {{ template "caraml-authz.postgresql.dsn" . }}
         resources:
 {{- toYaml .Values.deployment.resources | nindent 10 }}
+      {{- if .Values.caramlAuthzExternalPostgresql.enableProxySidecar }}
+      {{- tpl (toYaml (index .Values.caramlAuthzExternalPostgresql.sidecarSpec .Values.caramlAuthzExternalPostgresql.proxyType "spec")) . | nindent 6 }}
+      {{- end }}
+      {{- if .Values.serviceAccount.create }}
+      serviceAccountName: {{ default "mlp-authz" .Values.serviceAccount.name }}
```

Comment:
@maniteja6799 same here.

Reply:
thanks 👍 , Missed it earlier. Will change.